collections/_infographics/emissions/fossil-fuels-emissions.md

```diff
@@ -1,16 +1,60 @@
+* The graph shows the several-year **decline in consumption after 1979**, caused by the [oil crisis](https://en.wikipedia.org/wiki/1979_oil_crisis) after the revolution in Iran and the subsequent war between Iran and Iraq. At that time, world production fell by about 4%, and oil prices doubled.
+
+### Gas
+
+* **Usage:** Electricity and heat production in households (heating, cooking, water heating) and industry.
+* **Extraction and transportation:** In total, [4000 billion m<sup>3</sup>](https://www.bp.com/content/dam/bp/business-sites/en/global/corporate/pdfs/energy-economics/statistical-review/bp-stats-review-2019-natural-gas.pdf) were extracted in 2019, i.e. about 3.2 billion tons of natural gas. The largest producers include the USA (21% of world production), Russia (17%) and the Gulf States (together 18%). [Transportation of natural gas](https://en.wikipedia.org/wiki/Natural_gas#/media/File:Global_Gas_trade_both_LNG_and_Pipeline.png) occurs through pipelines and ships carrying liquefied gas.
+* Burning natural gas is significantly more efficient than burning coal for generating electricity. <glossary id="ipcc">IPCC</glossary> estimates the emission intensity of natural gas in the range of 410-650 g CO<sub>2</sub>eq/kWh (including extraction and transport); for coal combustion, it indicates a range of 740-910 g CO<sub>2</sub>eq/kWh. This fact represents an important reason for the so-called <glossary id="phase-out">coal phase-out</glossary>, which occurs when a combination of renewable energy sources and gas-fired power plants replaces coal-fired power plants.
+* In addition to CO<sub>2</sub> emissions from combustion, natural gas extraction and transport also produces methane emissions. These are mainly [gas leaks during mining](https://en.wikipedia.org/wiki/Fugitive_emission) estimated at around [100 million tons of CH<sub>4</sub>](https://www.carbonbrief.org/scientists-concerned-by-record-high-global-methane-emissions) per year, equivalent to about a third of anthropogenic methane emissions.
+
+### Cement
+{% comment %}
+{% include preview-box.html
+    title="More details on emissions from cement production"
+    text="The production of one tonne of cement creates approximately 1 to 1.2 tonnes of CO₂. Our explainer describes this process in more detail."
+    slug="emise-vyroba-cementu"
+%}
+{% endcomment %}
+
+* Heating a mixture of limestone and clay to temperatures of around 1450 °C produces cement. Additionally, the heat causes the limestone to decompose into calcium oxide and carbon dioxide. Approximately half of the total CO<sub>2</sub> emissions from cement production arise from the CaCO<sub>3</sub> + heat ⟶ CaO + CO<sub>2</sub> chemical reaction. The other half of the total emissions relates to coal or gas combustion. In the category *cement*, only emissions from the chemical reaction are entered in the graph. The graph includes combustion emissions for the respective fossil fuels.
```

Comment:
Switch sentence subject for legibility: 
Cement is produced by heating a mixture of limestone and clay to temperatures of around 1450 °C. This heating also causes limestone to decompose into calcium oxide and carbon dioxide. Approximately half of the total CO<sub>2</sub> emissions from cement production arise from the CaCO<sub>3</sub> + heat → CaO + CO<sub>2</sub> chemical reaction. The other half of the total emissions relates to coal or gas combustion in production. In the graph, the category *cement* only shows emissions from the chemical reaction. The emissions related to coal and gas combustion in cement production are reported under the categories for those fossil fuels.